languages/php.rst

```diff
@@ -413,6 +413,31 @@ Settings defined as environment variables have precedence over the ones defined
 in ``.symfony.cloud.yaml``, which has precedence over the ones defined in
 ``php.ini``.
 
+.. caution::
+
+   There's no limit on what you can define in your PHP configuration, but many
+   settings can break your application. This is a feature for advanced users.
+
+Default ``php.ini`` values
```

Comment:
Why is is needed? The php.ini file is easily editable from the project itself. Duplicating the info here means that we need to keep it uptodate?

Reply:
This is the platform defaults (not the ones from the templates) and some users don't manage to find the information.

Reply:
Got it